Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -14,6 +14,8 @@
 #include <Engine/Renderer/RenderObject/RenderObject.hpp>
 #include <Engine/Renderer/RenderObject/RenderObjectManager.hpp>
 
+#define POINT_SCALE_FACTOR 1_ra / 500
```

Comment:
Prefer a constexpr rather than a macro. Radium is (mainly) modern C++, not C.
constexpr are typed expression, macro are not !
```suggestion
constexpr Scalar POINT_SCALE_FACTOR = 1_ra / 500;
```